### PR TITLE
docs: description to customize the primer bed file

### PR DESCRIPTION
**Pull request by AKBrueggemann**
# Description

 Added the description how to use a customized primer bed file for the workflow to the documentation

## Checklist

- [x] I've formatted the PR title in accordance with the [structure of the conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/).
- [x] I've updated the code style using [`pre-commit`](https://ikim-essen.github.io/uncovar/dev-guide/contributing/#pre-commit) if needed.
- [x] I've updated or supplemented the documentation as needed.
- [x] I've read the [`CODE_OF_CONDUCT.md`] document.
- [x] I've read the [`CONTRIBUTING.md`] guide.

[`CODE_OF_CONDUCT.md`]: https://github.com/IKIM-Essen/uncovar/blob/master/CODE_OF_CONDUCT.md
[`CONTRIBUTING.md`]: https://github.com/IKIM-Essen/uncovar/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by thomasbtf** [comment]
@alethomas Could you please add you review?

File: docs/user-guide/configuration.md [hunk 1]
```diff
@@ -76,3 +76,7 @@ removed by specifying `nimagen-easy-seq`. A short example:
 | ----------- | ----------- | ----------- | ---------- | ---------------- | ---------- | ---------------- |
 | example-1   | PATH/TO/fq1 | PATH/TO/fq2 | 1970-01-01 | 0                | illumina   | revelo-rna-seq   |
 | example-2   | PATH/TO/fq  | PATH/TO/fq2 | 1970-01-01 | 1                | illumina   | nimagen-easy-seq |
+
+### Customize primer bed file
+
+The default primer file is a bed file from https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCoV-2019/V3. To use a custom primer file, you must save this file in a bed format under resources. Also, the path to this file must be changed in config.yaml. Go to the config folder and open config.yaml. In the "preprocessing" subcategory, change the path after "amplicon-primers" to the path where your primer file can be found.
```
Comment (thomasbtf) [inline docs/user-guide/configuration.md:82]:
```suggestion
When processing amplicon samples by default the ARTIC primers ([v3](https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCoV-2019/V3)) are removed. To use a custom primer file, you must save this file in a bed format under resources. Also, the path to this file must be changed in config.yaml. Go to the config folder and open config.yaml. In the "preprocessing" subcategory, change the path after "amplicon-primers" to the path where your primer file can be found.
```

Comment (thomasbtf) [inline docs/user-guide/configuration.md:82]:
```suggestion
The default primer file is a bed file from https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCoV-2019/V3. This primer clipping can be customized. First, the custom primers must be saved in bed format. Next, the path to this file must be changed in the config. Go to the config folder and open config.yaml. In the "preprocessing" subcategory, change the path after "amplicon-primers" to the path where your primer file can be found.
```

**Review by thomasbtf — COMMENTED** [review]
Just had another read. What do you think about this changes? :)

File: docs/user-guide/configuration.md (hunk 1 above)
Comment (alethomas) [reply docs/user-guide/configuration.md:82]: I vote for the 2nd suggestion

**Review by alethomas — COMMENTED** [review]
Need to rerun pre-commit

**Review by thomasbtf — APPROVED** [review]

**Pull request merged**